02-increase-cpuinfo-buffer-size.patch

```diff
@@ -0,0 +1,11 @@
+--- modules/NE10_init.c	2020-02-11 13:42:54.754327681 -0800
++++ modules/NE10_init_fixed.c	2020-02-11 13:43:47.201902486 -0800
```

Comment:
Is there a file with this name?

Reply:
Whoops, I pushed a proper diff, this should look better.